web/rest/user/config/services.yaml

```diff
@@ -55,3 +55,7 @@ services:
         resource: '../src/Controller/My/*'
         tags: ['controller.service_arguments']
 
+    Controller\Provider\:
+        resource: '../src/Controller/Provider/*'
+        tags: [ 'controller.service_arguments' ]
```

Comment:
Is it mandatory to create a new folder for one controller?

Reply:
I'm trying to align with the current status: this file also exists in the client portal, and I'd prefer to have both follow the same folder structure.

Additionally, I see this controller as the first one, not the only one ;)